### ISSUE TITLE
Ask for services offered by a bicycle shop

### ISSUE THREAD

**Issue by matkoniecz** [streetcomplete/StreetComplete#4012]
### General
Affected tag(s) to be modified/added: [service:bicycle:retail](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dbicycle)  [service:bicycle:repair](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dbicycle) [service:bicycle:rental](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dbicycle) ...
Question asked: **Mark services offered by this bicycle shop** / **Which services are offered by this bicycle shop?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [ ] 🐿️ Easily answerable by everyone from the outside but a survey is necessary - may be necessary to disable it by default
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation

**Element selection:** 
Skip objects where all (or any, or most) of this objects is tagged and all objects where any of this tags is present and was tagged recently
 

**Metadata needed:** 
None

**Proposed UI:**
Maybe resurvey with prefilled currently available things is a good idea?

**Comment by westnordost** [comment]
Shouldn't this be different questions for different properties?

**Comment by matkoniecz** [comment]
> Shouldn't this be different questions for different properties?

It is also possible, though answering multiple such question in row could be annoying compared to having a single question, especially as research needed for them is very similar.

On the other hand someone may prefer to answer only some of such question and keep others disabled.

Though, surveying retail/service could be enabled as default as almost always surveyable from outside.

BTW, I just want to express my irritation than in OSM tagging `shop=bicycle` may be not selling bicycles at all. That is just silly.

**Comment by matkoniecz** [comment]
So in the end I think that multiple questions make sense - for consistency and because at least some can be enabled by default.

**Comment by matkoniecz** [comment]
So there would be:

Can you buy bicycles here? [`service:bicycle:retail`](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:retail) (enabled by default)

Can you repair bicycles here? [`service:bicycle:repair`](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:repair) (disabled by default?)

Can you rent bicycles here? [`service:bicycle:rental`](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:rental) (disabled by default? Maybe with option "Primarily bicycle rental"?)

Is there a free bicycle pump which can be used by anybody? [`service:bicycle:pump`](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:pump) (disabled by default)

Are there are tools for do-it-yourself bike repair? [service:bicycle:diy](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:diy) (disabled by default)

Are there second hand bicycles sold here? [service:bicycle:second_hand](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:second_hand) (disabled by default, asked iff `service:bicycle:retail` was set to yes) Note that `yes` `no` `only` options are mentioned by OSM Wiki. Maybe [`second_hand=*`](https://wiki.openstreetmap.org/wiki/Key:second_hand) should be also set, but what about case when only second hand bicycles are sold and new parts are available? (disabled by default? Or maybe enabled?)

Can you wash bicycle here (for free or as a service)? [service:bicycle:cleaning](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:cleaning) (disabled by default)

**Comment by mnalis** [comment]
`service:bicycle:pump` is [already implemented](https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/air_pump/AddBicyclePump.kt) (and [enabled by default](https://github.com/streetcomplete/StreetComplete/issues/3053#issuecomment-1076823076)). 

Also, in my experience renting and repairing is usually advertised/visible from outside, so I'd lean towards enabling `service:bicycle:repair` and `service:bicycle:rental` by default. 

Although if rental is not one of the main focuses of shop owner, but it only has a bike or two on the side if someone comes, I guess it would be problematic to determine if not advertised. Over here, building with bicycles in it is usually either rent-only or sell-only (OTOH, repair shops here would be either repair-only or combined with selling bicycles; at about 50-50 rate)

**Comment by matkoniecz** [comment]
> Although if rental is not one of the main focuses of shop owner, but it only has a bike or two on the side if someone comes, I guess it would be problematic to determine if not advertised.

Yes, I think that this one requires asking inside

**Comment by westnordost** [comment]
Of all the different values, I only find [service:bicycle:repair](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:repair) to be really valuable. 

For the others, the **💤 no spam** rule kind of does not apply: The vast majority of bicycle shops sell bicycles and those that do not (e.g. only offer repairs and parts) are likely already tagged that way from the beginning. Similarily, the vast majority do not offer rental services, are not DIY workshops and do not offer cleaning (specifically).

`second_hand` may be good too, but usually requires to go inside and enquire. Might be used to sneakily also tag [service:bicycle:retail](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:retail) through options "Yes", "Only", "No", "Doesn't sell bicycles at all". Though, to be honest, if a `shop=bicycle` does not sell bicycles, I think a note that explains just what kind of shop this is exactly would be better, so all the eligible tags can be set at the same time.

In a nutshell, no objections against:
- asking whether they do repairs
- asking whether they also sell refurbished bicycles

**Comment by matkoniecz** [comment]
> Though, to be honest, if a shop=bicycle does not sell bicycles, I think a note that explains just what kind of shop this is exactly would be better

Note that in current OSM tagging place offering solely bicycle repair is also `shop=bicycle`

> Might be used to sneakily also tag [service:bicycle:retail](https://wiki.openstreetmap.org/wiki/Key:service:bicycle:retail) through options "Yes", "Only", "No", "Doesn't sell bicycles at all".

I like this idea.
